crates/store/re_chunk/src/slice.rs

```diff
@@ -450,13 +450,43 @@ impl Chunk {
                 .map(|(&component_name, list_array)| {
                     (
                         component_name,
-                        ListArray::new_empty(list_array.data_type().clone()),
+                        ArrowListArray::new_empty(list_array.data_type().clone()),
                     )
                 })
                 .collect(),
         }
     }
 
+    /// Removes all component columns from the [`Chunk`].
+    ///
+    /// The result is a new [`Chunk`] with the same number of rows and the same index columns, but
+    /// no components.
+    ///
+    /// WARNING: the returned chunk has the same old [`crate::ChunkId`]! Change it with [`Self::with_id`].
+    #[must_use]
+    #[inline]
+    pub fn components_removed(self) -> Self {
```

Comment:
`chunk.without_components()` reads more intuitively to me but I don't feel strongly